Jenkinsfile

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent {
+    label 'huge && bionic && x86_64 && !xenial && !trusty'
+  }
+
+  options {
+    timestamps()
+    timeout(time: 2, unit: 'HOURS')
+    quietPeriod(5)
```

Comment:
@djgalloway this is in the ceph-pull-request job. Is it required for anything specific?

Reply:
I don't think so.  5 seconds seems kind of silly anyway.  Maybe just comment it with an explanation for now.

Reply:
Removed since I guess what we really wanted is 'cancel-on-update' behaviour, rather than lazy builds...